docs/_layouts/tutorial_frame.html

```diff
@@ -19,7 +19,7 @@
 			margin: 0;
 		}
 		#map {
-			width: 600px;
+			width: 100%;
```

Comment:
If an initial width of 600px is desired (keeping `width: 600px` as currently set) then `max-width: 100%` could be used additionally to accomplish responsiveness of the maps.

Reply:
The default width was 600px so i thought we wanted to keep it that way and from then making it smaller. I changed the width to your proposal, the desired result is still the same so its cool :)